src/components/UserProfile/common.js

```diff
@@ -0,0 +1,35 @@
+import { trackUserInteraction, TRACKING_EVENTS } from 'services/analyticsTracking';
```

Comment:
I'm not conviced by that file name. Can't understand what is it purpose.

Why not something like TrakingHandler or something like that?